actions/tag/create_tag.go

```diff
@@ -0,0 +1,43 @@
+package tag
```

Comment:
Every other place we are using package name in plural form so `tags` will better.